services/src/test/scala/cromwell/pipeline/service/ProjectServiceTest.scala

```diff
@@ -48,6 +47,22 @@ class ProjectServiceTest extends AsyncWordSpec with Matchers with MockitoSugar w
       }
     }
 
+    "updateProjectVersion" should {
+      "return correct version of project" taggedAs Service in {
+        val projectId = ProjectId("projectId")
+        val userId = UserId.random
+        val version = PipelineVersion("v1.0.0")
+
+        when(projectRepository.getProjectById(projectId)).thenReturn(Future(Some(dummyProject.copy(version = version))))
+
+        projectService.updateProjectVersion(version, projectId, userId)
+        projectRepository.getProjectById(projectId).map {
+          case Some(project) => project.version shouldBe version
+          case None          => fail()
```

Comment:
This test tests nothing.
`projectRepository` is a mock that you set to return specific value. And here you are directly calling this mock to check that mock returns exactly what you set it to return. No `ProjectService` testing here.
If you modify `ProjectService.updateProjectVersion` to fail on any request, then this test will pass regardless.